gvr-renderableview/src/org/gearvrf/sample/renderableview/MainActivity.java

```diff
@@ -0,0 +1,63 @@
+/* Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gearvrf.sample.renderableview;
+
+import org.gearvrf.GVRActivity;
+import org.gearvrf.GVRScript;
+import org.gearvrf.scene_objects.view.GVRFrameLayout;
+import org.gearvrf.scene_objects.view.GVRView;
+
+import android.graphics.Color;
+import android.os.Bundle;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup.LayoutParams;
+import android.widget.Button;
+import android.widget.DatePicker;
+import android.widget.FrameLayout;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+public class MainActivity extends GVRActivity {
+    private GVRScript mScript;
+
+    private GVRFrameLayout mFrameLayout;
+    Button mButton;
+    DatePicker mDatePicker;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        createView();
+
+        mScript = new MainScript(this);
+        setScript(mScript, "gvr_note4.xml");
+    }
+
+    private void createView() {
+        mFrameLayout = new GVRFrameLayout(this);
+        mFrameLayout.setLayoutParams(new FrameLayout.LayoutParams(
+        		LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT));
+        mFrameLayout.setBackgroundColor(Color.CYAN);
+
+        View.inflate(this, R.layout.activity_main, mFrameLayout);
+    }
+
+    public GVRView getTextView() {
```

Comment:
this should probably be renamed to getFrameLayout()?  getTextView seems a little misleading.